vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -1053,6 +1053,15 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           return { key, columnId };
         });
 
+        grid.preventContextMenu =  tryCatchWrapper(function (event) {
```

Comment:
nit: extra whitespace.

```suggestion
        grid.preventContextMenu = tryCatchWrapper(function (event) {
```